src/components/Header/index.js

```diff
@@ -25,7 +25,7 @@ function Header() {
         <Link to="/develop" color="neutral-4">
           Develop
         </Link>
-        <Link to="/blog" color="neutral-4">
+        <Link to="/blogs" color="neutral-4">
```

Comment:
The existing portal is just /blog so wouldn't we want to keep it /blog so we don't break peoples links? #22

Reply:
ill change it back. for some reason i thought it was blogs on the old dev portal.